types/fs-extra/index.d.ts

```diff
@@ -207,9 +207,13 @@ export function rmdir(path: string | Buffer): Promise<void>;
 export function stat(path: string | Buffer, callback: (err: NodeJS.ErrnoException, stats: Stats) => any): void;
 export function stat(path: string | Buffer): Promise<Stats>;
 
-export function symlink(srcpath: string | Buffer, dstpath: string | Buffer, type: fs.symlink.Type | undefined, callback: (err: NodeJS.ErrnoException) => void): void;
+export function symlink(srcpath: string | Buffer, dstpath: string | Buffer, type: symlink.Type | undefined, callback: (err: NodeJS.ErrnoException) => void): void;
 export function symlink(srcpath: string | Buffer, dstpath: string | Buffer, callback: (err: NodeJS.ErrnoException) => void): void;
-export function symlink(srcpath: string | Buffer, dstpath: string | Buffer, type?: fs.symlink.Type): Promise<void>;
+export function symlink(srcpath: string | Buffer, dstpath: string | Buffer, type?: symlink.Type): Promise<void>;
+
+export namespace symlink {
```

Comment:
Why a namespace and not just a type? Seems like an overkill

Reply:
@alan-agius4 Because a [`SymlinkType`](https://github.com/KSXGitHub/DefinitelyTyped/blob/f969724802da1a2a4544dbda15617f4fac2a6262/types/fs-extra/index.d.ts#L267) already exists, I don't want to mess with it.